--- a/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/dhd_linux.c	2025-05-23 05:51:44.850000000 +0000
+++ b/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/dhd_linux.c	2025-05-23 05:53:38.990000000 +0000
@@ -349,6 +349,7 @@
 static void dhd_hang_process(struct work_struct *work_data);
 #endif /* OEM_ANDROID */
 MODULE_LICENSE("GPL and additional rights");
+MODULE_FIRMWARE("bcmdhd/nvram_ap6275p.txt");
 
 #if defined(MULTIPLE_SUPPLICANT)
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 25))